common/common-ui/src/main/java/com/duckduckgo/common/ui/view/text/DaxTextInput.kt

```diff
@@ -368,6 +369,8 @@ class DaxTextInput @JvmOverloads constructor(
             binding.internalEditText.keyListener = DigitsKeyListener.getInstance("0123456789.")
         } else if (inputType == INPUT_TYPE_MULTI_LINE || inputType == INPUT_TYPE_FORM_MODE) {
             binding.internalEditText.inputType = EditorInfo.TYPE_CLASS_TEXT or EditorInfo.TYPE_TEXT_FLAG_MULTI_LINE
+        } else if (inputType == INPUT_TYPE_URL_MODE) {
```

Comment:
Is this from a different PR?

Reply:
It's from this one, it's needed for the new maestro test, otherwise autocorrect sometimes removes hyphens in the URL